### ISSUE TITLE
Sonarqube temp directory not cleaned up after analysis

### ISSUE THREAD

**Issue by lneumeier** [Riverside-Software/sonar-openedge#632]
Problem:

Sonarqube tries to delete the temp directory from the computing engine after the analysis finished. Unfortunately, this fails with some locking issues.

```
2019.01.08 02:20:57 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.CeWorkerImpl] Execute task | project=v410.67 | type=REPORT | id=AWgrDArSRLqFT9EizTyL | submitter=admin
2019.01.08 02:24:25 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Extract report | status=SUCCESS | time=208332ms
2019.01.08 02:24:25 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Persist scanner context | status=SUCCESS | time=15ms
2019.01.08 02:24:25 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Propagate analysis warnings from scanner report | status=SUCCESS | time=16ms
2019.01.08 02:24:26 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Execute DB migrations for current project | status=SUCCESS | time=375ms
2019.01.08 02:24:26 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Generate analysis UUID | status=SUCCESS | time=0ms
2019.01.08 02:24:26 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Load analysis metadata | status=SUCCESS | time=32ms
2019.01.08 02:24:26 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Initialize | status=SUCCESS | time=0ms
2019.01.08 02:24:26 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Verify billing | status=SUCCESS | time=0ms
2019.01.08 02:24:33 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Build tree of components | components=27450 | status=SUCCESS | time=6968ms
2019.01.08 02:24:33 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Validate project | status=SUCCESS | time=94ms
2019.01.08 02:24:33 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Load quality profiles | status=SUCCESS | time=328ms
2019.01.08 02:24:33 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Load Quality gate | status=SUCCESS | time=15ms
2019.01.08 02:24:33 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Load new code period | status=SUCCESS | time=16ms
2019.01.08 02:24:34 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Detect file moves | reportFiles=25753 | dbFiles=25753 | addedFiles=0 | status=SUCCESS | time=687ms
2019.01.08 02:24:41 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Load duplications | duplications=43754 | status=SUCCESS | time=7453ms
2019.01.08 02:24:41 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Compute cross project duplications | status=SUCCESS | time=0ms
2019.01.08 02:24:48 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Compute size measures | status=SUCCESS | time=6532ms
2019.01.08 02:30:28 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Compute new coverage | status=SUCCESS | time=340581ms
2019.01.08 02:30:29 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Compute coverage measures | status=SUCCESS | time=140ms
2019.01.08 02:30:29 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Compute comment measures | status=SUCCESS | time=110ms
2019.01.08 02:30:29 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Copy custom measures | status=SUCCESS | time=15ms
2019.01.08 02:30:29 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Compute duplication measures | status=SUCCESS | time=328ms
2019.01.08 02:30:30 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Compute duplication data measures | status=SUCCESS | time=1375ms
2019.01.08 02:30:31 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Compute size measures on new code | status=SUCCESS | time=922ms
2019.01.08 02:30:31 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Compute language distribution | status=SUCCESS | time=78ms
2019.01.08 02:30:32 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Compute test measures | status=SUCCESS | time=360ms
2019.01.08 02:30:32 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Compute complexity measures | status=SUCCESS | time=156ms
2019.01.08 02:30:32 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Load measure computers | status=SUCCESS | time=0ms
2019.01.08 02:30:32 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Compute Quality Profile status | status=SUCCESS | time=16ms
2019.01.08 02:38:14 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Execute component visitors | status=SUCCESS | time=462290ms
2019.01.08 02:38:14 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Checks executed after computation of measures | status=SUCCESS | time=0ms
2019.01.08 02:38:18 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Compute measure variations | status=SUCCESS | time=4218ms
2019.01.08 02:38:18 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Compute Quality Gate measures | status=SUCCESS | time=16ms
2019.01.08 02:38:18 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Compute Quality profile measures | status=SUCCESS | time=0ms
2019.01.08 02:38:18 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Generate Quality profile events | status=SUCCESS | time=16ms
2019.01.08 02:38:18 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Generate Quality gate events | status=SUCCESS | time=0ms
2019.01.08 02:38:20 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Persist components | status=SUCCESS | time=1312ms
2019.01.08 02:38:20 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Persist analysis | status=SUCCESS | time=0ms
2019.01.08 02:38:20 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Persist analysis properties | status=SUCCESS | time=0ms
2019.01.08 02:38:20 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Persist measures | inserts=86 | status=SUCCESS | time=31ms
2019.01.08 02:52:14 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Persist live measures | insertsOrUpdates=1046949 | status=SUCCESS | time=833720ms
2019.01.08 02:52:14 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Persist new ad hoc Rules | status=SUCCESS | time=0ms
2019.01.08 02:56:36 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Persist issues | inserts=5 | updates=15740 | merged=0 | untouched=286337 | status=SUCCESS | time=262551ms
2019.01.08 02:56:36 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Persist project links | status=SUCCESS | time=0ms
2019.01.08 02:56:36 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Persist events | status=SUCCESS | time=16ms
2019.01.08 02:59:15 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Persist sources | status=SUCCESS | time=158503ms
2019.01.08 02:59:15 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Persist tests | status=SUCCESS | time=187ms
2019.01.08 02:59:15 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Persist cross project duplications | status=SUCCESS | time=0ms
2019.01.08 02:59:15 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Enable analysis | status=SUCCESS | time=109ms
2019.01.08 02:59:15 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Update last usage date of quality profiles | status=SUCCESS | time=16ms
2019.01.08 02:59:17 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Purge db | status=SUCCESS | time=2000ms
2019.01.08 03:01:10 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Index analysis | status=SUCCESS | time=113065ms
2019.01.08 03:01:10 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Send issue notifications | newIssuesNotifs=0 | newIssuesDeliveries=0 | myNewIssuesNotifs=0 | myNewIssuesDeliveries=0 | changesNotifs=0 | changesDeliveries=0 | status=SUCCESS | time=0ms
2019.01.08 03:01:10 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Publish task results | status=SUCCESS | time=0ms
2019.01.08 03:01:10 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.s.ComputationStepExecutor] Trigger refresh of Portfolios and Applications | status=SUCCESS | time=0ms
2019.01.08 03:02:43 INFO  ce[AWgrDArSRLqFT9EizTyL][o.s.c.t.CeWorkerImpl] Executed task | project=v410.67 | type=REPORT | id=AWgrDArSRLqFT9EizTyL | submitter=admin | status=SUCCESS | time=2506011ms
2019.01.08 03:49:17 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.CeWorkerImpl] Execute task | project=v500.68 | type=REPORT | id=AWgrXPC1RLqFT9EizTyM | submitter=admin
2019.01.08 03:52:29 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Extract report | status=SUCCESS | time=191878ms
2019.01.08 03:52:29 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Persist scanner context | status=SUCCESS | time=16ms
2019.01.08 03:52:29 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Propagate analysis warnings from scanner report | status=SUCCESS | time=15ms
2019.01.08 03:52:30 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Execute DB migrations for current project | status=SUCCESS | time=376ms
2019.01.08 03:52:30 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Generate analysis UUID | status=SUCCESS | time=0ms
2019.01.08 03:52:30 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Load analysis metadata | status=SUCCESS | time=31ms
2019.01.08 03:52:30 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Initialize | status=SUCCESS | time=0ms
2019.01.08 03:52:30 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Verify billing | status=SUCCESS | time=0ms
2019.01.08 03:52:36 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Build tree of components | components=27323 | status=SUCCESS | time=6719ms
2019.01.08 03:52:36 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Validate project | status=SUCCESS | time=93ms
2019.01.08 03:52:37 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Load quality profiles | status=SUCCESS | time=313ms
2019.01.08 03:52:37 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Load Quality gate | status=SUCCESS | time=16ms
2019.01.08 03:52:37 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Load new code period | status=SUCCESS | time=15ms
2019.01.08 03:52:38 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Detect file moves | reportFiles=25583 | dbFiles=25567 | addedFiles=16 | status=SUCCESS | time=688ms
2019.01.08 03:52:44 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Load duplications | duplications=43314 | status=SUCCESS | time=6969ms
2019.01.08 03:52:44 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Compute cross project duplications | status=SUCCESS | time=0ms
2019.01.08 03:52:51 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Compute size measures | status=SUCCESS | time=6219ms
2019.01.08 03:57:47 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Compute new coverage | status=SUCCESS | time=296599ms
2019.01.08 03:57:47 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Compute coverage measures | status=SUCCESS | time=125ms
2019.01.08 03:57:48 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Compute comment measures | status=SUCCESS | time=109ms
2019.01.08 03:57:48 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Copy custom measures | status=SUCCESS | time=16ms
2019.01.08 03:57:48 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Compute duplication measures | status=SUCCESS | time=297ms
2019.01.08 03:57:49 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Compute duplication data measures | status=SUCCESS | time=1172ms
2019.01.08 03:57:50 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Compute size measures on new code | status=SUCCESS | time=875ms
2019.01.08 03:57:50 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Compute language distribution | status=SUCCESS | time=62ms
2019.01.08 03:57:50 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Compute test measures | status=SUCCESS | time=63ms
2019.01.08 03:57:50 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Compute complexity measures | status=SUCCESS | time=156ms
2019.01.08 03:57:50 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Load measure computers | status=SUCCESS | time=0ms
2019.01.08 03:57:50 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Compute Quality Profile status | status=SUCCESS | time=16ms
2019.01.08 04:04:47 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Execute component visitors | status=SUCCESS | time=416538ms
2019.01.08 04:04:47 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Checks executed after computation of measures | status=SUCCESS | time=0ms
2019.01.08 04:04:51 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Compute measure variations | status=SUCCESS | time=4188ms
2019.01.08 04:04:51 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Compute Quality Gate measures | status=SUCCESS | time=16ms
2019.01.08 04:04:51 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Compute Quality profile measures | status=SUCCESS | time=15ms
2019.01.08 04:04:51 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Generate Quality profile events | status=SUCCESS | time=0ms
2019.01.08 04:04:51 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Generate Quality gate events | status=SUCCESS | time=0ms
2019.01.08 04:04:52 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Persist components | status=SUCCESS | time=1297ms
2019.01.08 04:04:52 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Persist analysis | status=SUCCESS | time=0ms
2019.01.08 04:04:52 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Persist analysis properties | status=SUCCESS | time=0ms
2019.01.08 04:04:52 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Persist measures | inserts=86 | status=SUCCESS | time=15ms
2019.01.08 04:18:32 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Persist live measures | insertsOrUpdates=1045550 | status=SUCCESS | time=819827ms
2019.01.08 04:18:32 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Persist new ad hoc Rules | status=SUCCESS | time=0ms
2019.01.08 04:22:41 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Persist issues | inserts=24 | updates=17967 | merged=0 | untouched=279471 | status=SUCCESS | time=248552ms
2019.01.08 04:22:41 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Persist project links | status=SUCCESS | time=15ms
2019.01.08 04:22:41 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Persist events | status=SUCCESS | time=0ms
2019.01.08 04:24:55 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Persist sources | status=SUCCESS | time=134658ms
2019.01.08 04:24:56 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Persist tests | status=SUCCESS | time=172ms
2019.01.08 04:24:56 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Persist cross project duplications | status=SUCCESS | time=0ms
2019.01.08 04:24:56 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Enable analysis | status=SUCCESS | time=109ms
2019.01.08 04:24:56 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Update last usage date of quality profiles | status=SUCCESS | time=0ms
2019.01.08 04:24:58 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Purge db | status=SUCCESS | time=1938ms
2019.01.08 04:26:45 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Index analysis | status=SUCCESS | time=107688ms
2019.01.08 04:26:45 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Send issue notifications | newIssuesNotifs=0 | newIssuesDeliveries=0 | myNewIssuesNotifs=0 | myNewIssuesDeliveries=0 | changesNotifs=0 | changesDeliveries=0 | status=SUCCESS | time=0ms
2019.01.08 04:26:45 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Publish task results | status=SUCCESS | time=0ms
2019.01.08 04:26:45 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.s.ComputationStepExecutor] Trigger refresh of Portfolios and Applications | status=SUCCESS | time=0ms
2019.01.08 04:27:40 ERROR ce[AWgrXPC1RLqFT9EizTyM][o.s.a.u.i.DefaultTempFolder] Failed to delete temp folder
java.nio.file.FileSystemException: D:\Sonarqube\sonarqube-7.5\temp\ce\7645893994787788560\6162200045052811384\source-1488.txt: Der Prozess kann nicht auf die Datei zugreifen, da sie von einem anderen Prozess verwendet wird.

	at sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:86)
	at sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:97)
	at sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:102)
	at sun.nio.fs.WindowsFileSystemProvider.implDelete(WindowsFileSystemProvider.java:269)
	at sun.nio.fs.AbstractFileSystemProvider.deleteIfExists(AbstractFileSystemProvider.java:108)
	at java.nio.file.Files.deleteIfExists(Files.java:1165)
	at org.sonar.api.utils.internal.DefaultTempFolder$DeleteRecursivelyFileVisitor.visitFile(DefaultTempFolder.java:115)
	at org.sonar.api.utils.internal.DefaultTempFolder$DeleteRecursivelyFileVisitor.visitFile(DefaultTempFolder.java:110)
	at java.nio.file.Files.walkFileTree(Files.java:2670)
	at java.nio.file.Files.walkFileTree(Files.java:2742)
	at org.sonar.api.utils.internal.DefaultTempFolder.clean(DefaultTempFolder.java:97)
	at org.sonar.api.utils.internal.DefaultTempFolder.stop(DefaultTempFolder.java:106)
	at org.sonar.ce.task.projectanalysis.filesystem.ComputationTempFolderProvider.stop(ComputationTempFolderProvider.java:65)
	at org.picocontainer.DefaultPicoContainer.stopAdapters(DefaultPicoContainer.java:1048)
	at org.picocontainer.DefaultPicoContainer.stop(DefaultPicoContainer.java:803)
	at org.sonar.core.platform.ComponentContainer.stopComponents(ComponentContainer.java:165)
	at org.sonar.ce.task.container.TaskContainerImpl.close(TaskContainerImpl.java:88)
	at org.sonar.ce.task.projectanalysis.taskprocessor.ReportTaskProcessor.process(ReportTaskProcessor.java:83)
	at org.sonar.ce.taskprocessor.CeWorkerImpl.executeTask(CeWorkerImpl.java:137)
	at org.sonar.ce.taskprocessor.CeWorkerImpl.findAndProcessTask(CeWorkerImpl.java:100)
	at org.sonar.ce.taskprocessor.CeWorkerImpl.withCustomizedThreadName(CeWorkerImpl.java:84)
	at org.sonar.ce.taskprocessor.CeWorkerImpl.call(CeWorkerImpl.java:76)
	at org.sonar.ce.taskprocessor.CeWorkerImpl.call(CeWorkerImpl.java:46)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2019.01.08 04:27:44 INFO  ce[AWgrXPC1RLqFT9EizTyM][o.s.c.t.CeWorkerImpl] Executed task | project=v500.68 | type=REPORT | id=AWgrXPC1RLqFT9EizTyM | submitter=admin | status=SUCCESS | time=2306289ms
```

Is this a problem caused by the sonar-openedge plugin or is a sonarqube problem?

**Comment by gquerret** [comment]
The OpenEdge plugin doesn't have any extension on the Compute Engine side, so that's a plain SonarQube issue. You should try https://community.sonarsource.com/ 
Don't you have any anti-virus software running on the server ?

**Comment by lneumeier** [comment]
This problem will be fixed with Sonarqube 7.6
https://jira.sonarsource.com/browse/SONAR-11622
